Core/src/org/sleuthkit/autopsy/datasourceprocessors/VolatilityProcessor.java

```diff
@@ -336,7 +339,13 @@ private String normalizePath(String filePath) {
         filePath = filePath.replaceAll("\\\\", "/");
         filePath = filePath.toLowerCase();
         filePath = filePath.replaceAll("/systemroot/", "/windows/");
+        // catches 1 type of file in cmdline
+        filePath = filePath.replaceAll("%systemroot%", "/windows/");
         filePath = filePath.replaceAll("device/","");
+        // helps with finding files in handles plugin
+        if (filePath.contains("harddiskvolume[0-9]/")){
+            filePath = filePath.substring(filePath.indexOf("harddiskvolume[0-9]/") -1);
```

Comment:
Isn't this the same as the next line?

Reply:
Actually after looking at it again get rid of the if and the line after the if and keep the substring as this will be better for this case \Device\clfs\Device\HarddiskVolume2\Users\mark\AppData\Local\Microsoft\Windows\UsrClass.dat{e15d4b01-1598-11e8-93e6-080027b5e733}.TM

Reply:
OK.  Can you get rid of the next line then?